src/Microsoft.VisualStudio.Services.Agent/Util/VarUtil.cs

```diff
@@ -65,6 +65,22 @@ public static string OSArchitecture
             }
         }
 
+        /// <summary>
+        /// Returns value in environment variables format.
+        /// Example: env.var -> ENV_VAR
+        /// </summary>
+        /// <param name="value"></param>
+        /// <returns></returns>
+        public static string ConvertToEnvVariableFormat(string value)
+        {
+            if(string.IsNullOrEmpty(value)) // Should it check by IsNullOrWhiteSpace instead?
+            {
+                return string.Empty;
+            }
+
+            return value.Replace('.', '_').Replace(' ', '_').ToUpperInvariant();
```

Comment:
The entire method could be simplified this way
`return value?.Replace('.', '_').Replace(' ', '_').ToUpperInvariant() ?? string.Empty;`

Reply:
Changed, thanks